com.unity.shadergraph/Editor/Drawing/Views/MaterialGraphView.cs

```diff
@@ -21,6 +21,11 @@ public MaterialGraphView()
             deleteSelection = DeleteSelectionImplementation;
         }
 
+        protected override bool canCopySelection
+        {
+            get { return selection.OfType<Node>().Any() || selection.OfType<GroupNode>().Any() || selection.OfType<BlackboardField>().Any(); }
```

Comment:
Doesn't work on trunk, needs a conditional import of `UnityEditor.Graphs` if not `UNITY_2018_1`